services/provider/server/server.go

```diff
@@ -393,6 +393,10 @@ func (s *OCSProviderServer) getCephClientInformation(ctx context.Context, name s
 	return cephClient.Status.Info["secretName"], cephClient.Annotations[controllers.StorageCephUserTypeAnnotation], nil
 }
 
+func scrCephClientCsiSecretName(secretType, suffix string) string {
```

Comment:
Shouldn't this be called `storageClaimCepCSISecretName` mainly because it produces a name for a secret reconciled by the storage claim, not the request?

Reply:
Sure, done.